v21.1/crdb-internal.md

```diff
@@ -9,7 +9,7 @@ The `crdb_internal` [system catalog](system-catalogs.html) is a schema that cont
 {{site.data.alerts.callout_danger}}
 We do not recommend using `crdb_internal` tables in production environments for the following reasons:
 
-- The contents of `crdb_internal` are unstable, and subject to change in new releases of CockroachDB, without prior notice.
+- The contents of `crdb_internal` are unstable, and subject to change in new releases of CockroachDB, without notice.
```

Comment:
Same comment: this feels like legalese that should not be altered. Can we track down its origin?

Reply:
Reverted across all versions.